models/classes/validator/PropertyChangedValidator.php

```diff
@@ -33,6 +34,22 @@ public function isPropertyChanged(core_kernel_classes_Property $property, OldPro
             return true;
         }
 
+        $this->isPropertyTypeChanged($property, $oldProperty);
+        
+        $this->isRangeChanged($property, $oldProperty);
+
+        return false;
+    }
```

Comment:
Missing empty line separator

```suggestion
    }
    
```